src/main/java/com/google/sps/data/InsertAndUpdate.java

```diff
@@ -0,0 +1,53 @@
+package com.google.sps.data;
```

Comment:
I had a bunch of comments about these classes on the other PR. please address them.